pkgs/unified_analytics/example/unified_analytics_example.dart

```diff
@@ -19,27 +16,56 @@ final Analytics analytics = Analytics.development(
   dartVersion: 'Dart 2.19.0',
 );
 
+// Timing a process and sending the event
 void main() {
   DateTime start = DateTime.now();
-  print('###### START ###### $start');
 
-  // Confirm to analytics instance that the message was shown;
-  // simplified for this example, tools using this package will
-  // invoke this method after confirming they have showed the
-  // message
-  analytics.clientShowedMessage();
+  // Each client using this package will have it's own
+  // method to show the message but the below is a trivial
+  // example of how to properly initialize the analytics instance
+  if (analytics.shouldShowMessage) {
+    // Simulates displaying the message, this will vary from
+    // client to client; ie. stdout, popup in IDE, etc.
+    print(analytics.getConsentMessage);
+
+    // After receiving confirmation that the message has been
+    // displayed, invoking the below method will successfully
+    // onboard the tool into the config file and allow for
+    // events to be sent on the next creation of the analytics
+    // instance
+    //
+    // The rest of the example below assumes that the tool has
+    // already been onboarded in a previous run
+    analytics.clientShowedMessage();
+  }
+
+  print('Current is opted in: ${analytics.telemetryEnabled}');
```

Comment:
should this be "Current user"? something else?

Reply:
Yep, seems like GitHub might have lagged a bit but it should say "current user"